projects/Flarum/default.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+
+{
+  metadata = {
+    summary = "Flarum is a technically advanced, open and extensible discussion platform.";
+    subgrants = [
+      "Flarum"
+    ];
+  };
+
+  nixos.modules.programs = {
+    flarum = {
+      name = "flarum";
+      module = ./programs/Flarum/module.nix;
+      examples.basic = {
+        module = ./programs/Flarum/examples/basic.nix;
+        description = "";
+        tests.basic = null;
+      };
```

How about we just do this? Just don't forget to create the files

```suggestion
      module = ./module.nix;
      examples.basic = {
        module = ./programs-example.nix;
        description = "";
        tests.basic = null;
      };
```